modules/custom/d_p/d_p.module

```diff
@@ -67,6 +67,26 @@ function d_p_preprocess_paragraph(&$variables) {
    * @var \Drupal\field\Entity\FieldConfig $field
    */
   foreach ($paragraphFieldDefinitions as $fieldName => $fieldConfig) {
+    if ($fieldName == 'field_d_settings') {
+      $field = $paragraph->get($fieldName);
+
+      if ($field->isEmpty()) {
+        continue;
+      }
+
+      $settings = json_decode($field->first()->get('value')->getValue());
+
+      if (!isset($settings->custom_class) || !isset($settings->custom_theme_colors)
+        || strpos($settings->custom_class, 'theme-custom') === FALSE
+      ) {
+        continue;
+      }
+
+      $wrapper_attributes['style'] = [
+        'background-color: ' . $settings->custom_theme_colors->background . ';',
+        'color: ' . $settings->custom_theme_colors->text . ';',
+      ];
+    }
```

Comment:
jezeli tam w updacie, ma byc usuniete pole `field_d_background_color` to tu poniżej można usunąc jego obsługę - chyba nie będzie nam już potrzebna :thinking: